.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
       - ia-web-development
+      - ia-development
```

Comment:
# (Maybe?) Backend Bug

Path: `/octp/Users`
Logged in as user created with: `bin/create_staff_user.py -e joshua+csg-compact-ed@magicclay.io -f kravitz -g joshua -t compact-ed -c octp -j oh`

I added a new user. Then, I updated their permissions. I get back a 502 Bad Gateway response.

Request Body: 
```json
{
    "octp": {
        "actions": {
            "read": false,
            "admin": false
        },
        "jurisdictions": {
            "ar": {
                "actions": {
                    "write": false,
                    "admin": false
                }
            },
            "mt": {
                "actions": {
                    "write": false,
                    "admin": false
                }
            }
        }
    }
}
```

### Error message
Gateway logs–
```(7242f70d-a907-4440-bad3-caa5203d98d7) Lambda execution failed with status 200 due to customer function error: {'permissions': defaultdict(<class 'dict'>, {'octp': {'value': {'actions': ['Missing data for required field.']}}})}. Lambda request id: ea61a756-9304-4a6c-bdad-ef1708f7f21f```

Lambda logs–
```python
[ERROR] ValidationError: {'permissions': defaultdict(<class 'dict'>, {'octp': {'value': {'actions': ['Missing data for required field.']}}})}
Traceback (most recent call last):
  File "/opt/python/aws_lambda_powertools/logging/logger.py", line 450, in decorate
    return lambda_handler(event, context, *args, **kwargs)
  File "/opt/python/cc_common/utils.py", line 73, in caught_handler
    'body': json.dumps(fn(event, context), cls=ResponseEncoder),
  File "/opt/python/aws_lambda_powertools/logging/logger.py", line 450, in decorate
    return lambda_handler(event, context, *args, **kwargs)
  File "/opt/python/cc_common/utils.py", line 149, in authorized
    return fn(event, context)
  File "/var/task/handlers/users.py", line 64, in get_users
    resp['users'] = [user_api_schema.load(user) for user in users]
  File "/opt/python/marshmallow/schema.py", line 722, in load
    return self._do_load(
  File "/opt/python/marshmallow/schema.py", line 909, in _do_load
    raise exc
```

What follows is confusing, because it now seems like this API request fails with a gateway error even on a new user. Almost seems like the database is in a corrupt state.